utils/crawlGameSchedule.js

```diff
@@ -0,0 +1,54 @@
+const puppeteer = require("puppeteer");
+
+const crawlGameSchedule = async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  await page.goto("https://www.koreabaseball.com/Schedule/GameCenter/Main.aspx#none;");
```

Comment:
```suggestion
  await page.goto("https://www.koreabaseball.com/Schedule/GameCenter/Main.aspx#none");
```
주소 안에 세미콜론은 빼는게 맞나요??